### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by WilliamNgiam**
Initialize for UChicago workshop

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-longitude: "-1"    # decimal longitude of the workshop venue (use https://www.latlong.net)
-humandate: "FIXME"    # human-readable dates for the workshop (e.g., "Feb 17-18, 2020")
-humantime: "FIXME"    # human-readable times for the workshop (e.g., "9:00 am - 4:30 pm")
-startdate: FIXME      # machine-readable start date for the workshop in YYYY-MM-DD format like 2015-01-01
-enddate: FIXME        # machine-readable end date for the workshop in YYYY-MM-DD format like 2015-01-02
-instructor: ["FIXME"] # boxed, comma-separated list of instructors' names as strings, like ["Kay McNulty", "Betty Jennings", "Betty Snyder"]
-helper: ["FIXME"]     # boxed, comma-separated list of helpers' names, like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
-email: ["fixme@example.org"]    # boxed, comma-separated list of contact email addresses for the host, lead instructor, or whoever else is handling questions, like ["marlyn.wescoff@example.org", "fran.bilas@example.org", "ruth.lichterman@example.org"]
+venue: "John Crerar Library, University of Chicago"        # brief name of host site without address (e.g., "Euphoric State University")
```
Comment (angela-li) [inline index.md]: Perhaps note that this is the Zar Room?

File: index.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+country: "us"      # lowercase two-letter ISO country code such as "fr" (see https://en.wikipedia.org/wiki/ISO_3166-1#Current_codes)
+language: "en"     # lowercase two-letter ISO language code such as "fr" (see https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)
+latitude: "42"     # decimal latitude of workshop venue (use https://www.latlong.net/)
+longitude: "-87"    # decimal longitude of the workshop venue (use https://www.latlong.net)
+humandate: "Apr 16-17, 2020"    # human-readable dates for the workshop (e.g., "Feb 17-18, 2020")
+humantime: "8:30am - 4:30pm"    # human-readable times for the workshop (e.g., "9:00 am - 4:30 pm")
+startdate: 2020-04-16      # machine-readable start date for the workshop in YYYY-MM-DD format like 2015-01-01
+enddate: 2020-04-17        # machine-readable end date for the workshop in YYYY-MM-DD format like 2015-01-02
+instructor: ["Brooke Luetgert", "William Ngiam"] # boxed, comma-separated list of instructors' names as strings, like ["Kay McNulty", "Betty Jennings", "Betty Snyder"]
+helper: ["Angela Li"]     # boxed, comma-separated list of helpers' names, like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
```
Comment (angela-li) [inline index.md]: Also add Lachlan Deer, Karina Acosta as helpers

File: index.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+email: ["uccarpentries@gmail.com", "luetgert@uchicago.edu"]    # boxed, comma-separated list of contact email addresses for the host, lead instructor, or whoever else is handling questions, like ["marlyn.wescoff@example.org", "fran.bilas@example.org", "ruth.lichterman@example.org"]
 collaborative_notes:             # optional: URL for the workshop collaborative notes, e.g. an Etherpad or Google Docs document (e.g., https://pad.carpentries.org/2015-01-01-euphoria)
-eventbrite:           # optional: alphanumeric key for Eventbrite registration, e.g., "1234567890AB" (if Eventbrite is being used)
+eventbrite: "98439686753"          # optional: alphanumeric key for Eventbrite registration, e.g., "1234567890AB" (if Eventbrite is being used)
```
Comment (angela-li) [inline index.md:17]: Nice job!

**Review by angela-li — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (WilliamNgiam) [reply index.md]: Done!

**Pull request merged**